ERCS/erc-x.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Validation Module Extension for ERC-7579
+description: Introduces three new module types for signature validation and permissioning
+author: zeroknots (@zeroknots), Konrad Kopp (@kopy-kat), Taek Lee (@leekt), Fil Makarov (@filmakarov)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR: https://ethereum-magicians.org/c/ercs/57